CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 * Release published on GitHub now including changes from CHANGELOG. @NodeGuy
 * Added ESLint errors: no-undef and no-unused-vars. @NodeGuy
 * Modal when a node doesn't signal any new blocks @faboweb
+* Checking known nodes round robin and signaling an error if no compatible and online nodes are available @faboweb
```

Comment:
sorry @faboweb i don't understand this sentence 😢

Reply:
changed it